pom.xml

```diff
@@ -509,6 +509,12 @@
       </dependency>
 
       <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>jsr305</artifactId>
+        <version>1.3.9</version>
+      </dependency>
```

Comment:
don't we also have to add this dependency to every module utilizing `@Nullable` and friends?